README.md

```diff
@@ -45,7 +45,7 @@ Consider also ignoring the `cypress` folder if you are not running tests.
 
 #### 4. Running
 
-The `vercel` command will build and deploy your project for you.
+The `vercel` command will build and deploy your project for you. You must [download Vercel](https://vercel.com/download) by running `npm i -g vercel` before you run this command for the first time
```

Comment:
This would have me visiting the webpage looking for a download first. I don't think it's neccessary to go into this much detail. People who use JS/Node are familiar with the concept of installing a node package, so I'd just keep it to "globally installing the Vercel CLI" with `npm i -g vercel`

Reply:
I also felt that "download" was not the most accurate so changed it here: https://github.com/thgh/now-sapper/commit/96d36a04fa434a5669674272cc350d91243c5c7e#diff-04c6e90faac2675aa89e2176d2eec7d8R48

Reply:
Sounds good to me